Java_Beginners/59_javaBeginners.java

```diff
@@ -0,0 +1,43 @@
+// apples.java file
+
+class apples{
```

Comment:
Please start names of class from capital letter

Reply:
Done. More of these to follow